packages/core/test.setup.js

```diff
@@ -0,0 +1,7 @@
+/* globals module require globalThis */
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { MemoryStorage } = require('@medplum/core');
+
+module.exports = () => {
+  Object.defineProperty(globalThis.window, 'sessionStorage', { value: new MemoryStorage() });
```

Comment:
Out of curiosity, why was this necessary?

Reply:
sessionStorage does not exist in the jsdom environment

Reply:
I think we weren't actually testing `sessionStorage` before